src/Data/Mapper/PositionMapper.php

```diff
@@ -27,4 +27,24 @@ public function getPositions($options = [])
 
         return [$this->mapObjects($result), $paginate];
     }
+
+    protected function mapObjects(array $records)
```

Comment:
Mapping objects I believe should be in mapper, instead of the entity. Plus, it makes us be explicit about what is contained in the entity.